install.sh

```diff
@@ -1,35 +0,0 @@
-#!/bin/sh
```

Comment:
Did you forget to commit the file? I  see it's removed, but not added anywhere

Reply:
good catch! I was rushing.